oauth2server/proj/settings/default.py

```diff
@@ -81,4 +81,128 @@
         'rest_framework.authentication.BasicAuthentication',
         'rest_framework.authentication.SessionAuthentication',
     )
-}
\ No newline at end of file
+}
+
+
+# Custom logging to give us unique logging to work on remote containers and normal logging
```

Comment:
OK. Removed commented out code. I've also updated comments on what loggers are in place.